lib/hidapi/bld.bat

```diff
@@ -0,0 +1,23 @@
+mkdir build
+cd build
+
+python -c "print(r'%PREFIX%'.replace('\\','/'))" > temp.txt
+set /p BASH_PREFIX=<temp.txt
+
+set MINGW=%MINGW64%
+set PATH=%MINGW%;%PATH%
+
+cmake ^
+	-G "MinGW Makefiles"^
+    -DHIDAPI_WITH_HIDRAW="FALSE"^
+    -DHIDAPI_WITH_LIBUSB="TRUE"^
```

Comment:
i destroyed formatting here, sorry. Please adjust

Reply:
Also, cmake reports not using these variables. This might be a problem

Reply:
Those are windows specific variables. I deleted them.